tests/e2e/tests/dashboard/dashboard_test.go

```diff
@@ -384,6 +384,19 @@ func citadelQueryFilterFn(queries []string) []string {
 		if strings.Contains(query, "secret_deleted_cert_count") {
 			continue
 		}
+		if strings.Contains(query, "server_csr_count") {
```

Comment:
if you didn't add any new metrics why did you have to add more here?

Reply:
I'm looking into that now. I believe this PR was added before the `dashboard_test` was merged in, so before I changed it today, all the tests were passing.

Reply:
Confirmed that these metrics are exported / can be triggered and read, but have no data points ingested in this e2e test. Turns out due to prefixing issues, this test should actually fix a few of the dashboard panels (adding `citadel_server` prefix back as the dashboard expects).

At the moment, unfortunately, so many of the Citadel dashboard metrics are ignored that the test isn't covering much of anything save for the process metrics.

Reply:
Found the difference. When we use a `promtheus.NewCounterVec`, it seems to initially use `0` as the count for the metric, whereas for `monitoring.NewSum`, it shows up as `no data`. That's why these metrics got past the tests initially.